libs/libcommon/src/libcommon/viewer_utils/asset.py

```diff
@@ -27,6 +28,13 @@ def create_asset_dir(
     return dir_path, url_dir_path
 
 
+def create_file_dir(dataset: str, config: str, split: str, assets_directory: StrPath) -> Tuple[Path, str]:
```

Comment:
maybe rename this one into `create_split_asset_dir`, and the previous one (`create_asset_dir`) into `create_cell_asset_dir`.

Reply:
maybe both could use a common function with optional parameters, to avoid code duplication